### PR TITLE
Add unit test for the EventStorageService

### PR DESCRIPTION
**Pull request by WengerK**
Here some basic tests.

They serve as Example for futur tests or as "code" example for developers which discover the module.
As I always say, people should read the tests to understand the module - thoses tests should help them - especially Units ones.

We still have to works on heavy tests (and more valuable for the module stability himself) as:

- TrackCheckoutStepEvent (see issue #25 - done by #30)
- EventStorageService (done in this PR #18 )
- EventTrackerService (done by #20)
- CommerceEventsSubscriber (done by #28)
- Product  (see issue #21 )
- EventsController  (see issue #22 - done by #31 )
- commerce_google_tag_manager_commerce_product_view  (see issue #24  - done by #29)

But let's merge first this in dev & then works separately on each other cases

### CODE REVIEW

File: tests/src/Unit/AlterEventDataEventTest.php [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+      'Purchase' => [
+        [
+          'event' => EventTrackerService::EVENT_PURCHASE,
+          'ecommerce' => [
+            'purchase' => [
+              'actionField' => [
+                'id' => '1',
+                'affiliation' => 'Commerce Website',
+                'revenue' => '11.99',
+                'shipping' => '0',
```
Comment (WengerK) [inline tests/src/Unit/AlterEventDataEventTest.php]:
```suggestion
```

Remove. Shipping should be tests in a closed tests - alone.

File: tests/src/Unit/AlterEventDataEventTest.php [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+                'coupon' => '',
```
Comment (WengerK) [inline tests/src/Unit/AlterEventDataEventTest.php]:
```suggestion
```

Remove. Coupon should be tests in a closed tests - alone.

File: tests/src/Kernel/EventStorageServiceTest.php [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    ], $result);
+  }
+
+  /**
+   * @covers ::addEvent
+   *
+   * Asserts the events queue follows the FIFO (First in First out) pattern.
+   */
+  public function testAddEventFifoQueue() {
+    $this->testAddEvent();
```
Comment (WengerK) [inline tests/src/Kernel/EventStorageServiceTest.php]:
```suggestion
```
useless

File: tests/src/Kernel/EventStorageServiceTest.php [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    ], $events);
+  }
+
+  /**
+   * @covers ::addEvent
+   *
+   * Asserts strictly same event aren't added twice in the events queue.
+   */
+  public function testAddEventSameSkipped() {
+    $this->testAddEvent();
```
Comment (WengerK) [inline tests/src/Kernel/EventStorageServiceTest.php]:
```suggestion
```
useless

**Review by wanze — APPROVED** [review]
Thanks, this looks good! Please merge, I hope you have enough permissions - otherwise let me know :)

**Pull request merged**